browser/js/signup/signup.html

```diff
@@ -1,42 +1,55 @@
-<section id='signup-form'>
-    <h1>Sign Up</h1>
-    <div>
-        <alert type="danger" ng-show="error">
-            {{ error }}
-            <br>
-        </alert>
-        <form name="myForm">
-            <div class="form-group">
-                <label>First Name:</label>
-                <input type="text" class="inputColor form-control" ng-model="user.firstName" required/>
-            </div>
-            <div class="form-group">
-                <label>Last Name:</label>
-                <input type="text" class="inputColor form-control" ng-model="user.lastName" required/>
-            </div>
-            <div class="form-group">
-                <label>Your Email:</label>
-                <input type="email" class="inputColor form-control" ng-model="user.email" required/>
-            </div>
-            <div class="form-group">
-                <label>Your Password:</label>
-                <input type="password" class="inputColor form-control" ng-model="user.password" required/>
+<div class="container">
```

Comment:
I know see what you mean about the section ids -- I noticed that not all the scss pages have corresponding ids though -- is this something we should discuss adding in on Monday?